src/hooks/useWindowDimensions.ts

```diff
@@ -1,5 +1,12 @@
 import { useState, useEffect, useCallback } from 'react'
 
+/**
```

Comment:
Why are we deprecating it and not just removing it?

Reply:
a new PR will be made removing it. I just didn't remove it to avoid making this PR become really large